frontend/packages/console-shared/src/components/quick-starts/QuickStartsCatalogCard.tsx

```diff
@@ -50,6 +52,10 @@ const QuickStartsCatalogCard: React.FC<QuickStartsCatalogCardProps> = ({
   );
   const [isOpen, setOpen] = React.useState<boolean>(false);
 
+  React.useEffect(() => {
+    setShowQSTile && setShowQSTile(showTile);
```

Comment:
why not add this line inside `onRemove`

```suggestion
    shouldShowQSTile && shouldShowQSTile(false);
```

Reply:
update

Reply:
Well, `onRemove` will only be fired on button click.
So in a situation, where the Tile is already in a removed state, the `showTile` in `Child` is set to `false` but the `shouldShowTile` is set to `true` in parent as `onRemove` was not called.

The idea here is to keep `shouldShowTile` in parent and `showTile` in Child to be in sync